build/templates/_converters.py.mako

```diff
@@ -29,6 +29,52 @@ def convert_timedelta_to_microseconds(value, library_type):
     return _convert_timedelta(value, library_type, 1000000)
 
 
+# This converter is not called from the normal codegen path for function. Instead it is
+# call from init and is a special case. Also, it just returns a string rather than a ctype object
+def init_with_options_converter(value, encoding):
+    if type(value) is str:
+        init_with_options_string = value
+    else:
+        good_keys = {
+            'rangecheck': 'RangeCheck',
```

Comment:
Why allow multiple spellings? We don't typically do that elsewhere.

Reply:
I wanted to support either option strings they already have, or an option string that matches Python naming conventions.

Reply:
"option strings they already have" would be given by customers as str and passed right along by the Python bindings, no?

Reply:
I think so, the thinner the wrappers the better off we will be.